internal/usage/usage.go

```diff
@@ -311,4 +311,15 @@ Valid values: KEYS|IAM_ROLE.`
 Valid values: 15|30|60.`
 	SnapshotIntervalHours = `Number of hours between snapshots.
 Valid values: 6|8|12|24.`
+	LiveMigrationDestinationClusterName      = "Human-readable label that identifies the Atlas destination cluster."
+	LiveMigrationDestinationProjectID        = "Human-readable label that identifies the Atlas destination project."
+	LiveMigrationHostEntries                 = "List of hosts running the MongoDB Agent that can transfer your MongoDB data from the source (Cloud Manager or Ops Manager) to destination (Atlas) deployments. Each live migration process uses its own dedicated migration host."
+	LiveMigrationSourceClusterName           = "Human-readable label that identifies the source Cloud Manager or Ops Manager cluster."
+	LiveMigrationSourceProjectID             = "Unique 24-hexadecimal digit string that identifies the source project."
+	LiveMigrationSourceSSL                   = "Flag that indicates whether you have TLS enabled."
```

Comment:
```suggestion
	LiveMigrationSourceSSL                   = "Flag that indicates whether data source has TLS enabled."
```
(to be approved by docs)